### PR TITLE
create landing-page

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #28

### CODE REVIEW

**Review by yosefanajjar — DISMISSED** [review]

File: src/routes/App/index.js [hunk 1]
```diff
@@ -3,11 +3,7 @@ import 'antd/dist/antd.css';
 import './style.css';
```
Comment (Fatmasiam) [inline src/routes/App/index.js]: Why this file here?

File: src/containers/Landing/landing.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    border-color: #E9663A;
+    text-shadow: none;
+    box-shadow: none;
+}
+
+.ant-btn-primary:hover, .ant-btn-primary:focus {
+    color: #fff;
+    background-color: #E9663A;
+    border-color: #E9663A;
+}
```
Comment (Fatmasiam) [inline src/containers/Landing/landing.css:108]: add new line :)

Comment (AlaaTaima) [reply src/containers/Landing/landing.css:108]: done

**Review by fares98 — DISMISSED** [review]

File: src/containers/Landing/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button } from 'antd';
+import { ReactComponent as Illustration } from '../assets/images/landing.svg';
+import './landing.css';
+
+const LandingPage = () => {
+  return (
+    <div>
+      <Illustration className="Illustration" />
+      <div className="landingPAge">
```
Comment (alaa-yasin) [inline src/containers/Landing/index.js]: I am sure the 'A' in the className (landingPAge) is by mistake :)

**Review by alaa-yasin — DISMISSED** [review]

**Comment by Fatmasiam** [comment]
check the accessibility and IPad view

File: src/containers/Landing/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <p className="landingPage__title">
```
Comment (Fatmasiam) [inline src/containers/Landing/index.js]:
```suggestion
        <p className="landing__title">
```

File: src/containers/Landing/index.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          Practice the art of good mental health
+        </p>
+        <p className="landingPage__content">
+          Tell your story and positively guide your thoughts
+        </p>
+        <Button type="primary" className="landingPage__button">
+          Sign In
+        </Button>
+        <p className="landingPage__signUpLink">Create a new account</p>
```
Comment (Fatmasiam) [inline src/containers/Landing/index.js]:
```suggestion
        <p className="landing__link">Create a new account</p>
```

File: src/containers/Landing/landing.css [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
```
Comment (Fatmasiam) [inline src/containers/Landing/landing.css:79]: Look to line 24,34, 43, and 54 
```css
 font-size: 14px;
```
you can collect all the repeated CSS and put it at the top with each other

Comment (Fatmasiam) [reply src/containers/Landing/landing.css:79]: also, this is repeated
```CSS 
font-family: Source Sans Pro;
font-family: Open Sans;
``` 

and i don't know we are using a lot of fonts !!!!

File: src/containers/Landing/index.js (hunk 3 above)
Comment (AlaaTaima) [reply src/containers/Landing/index.js]: yeah (:

**Review by fares98 — DISMISSED** [review]

**Review by fares98 — DISMISSED** [review]

**Review by fares98 — DISMISSED** [review]

**Review by fares98 — DISMISSED** [review]

**Review by alaa-yasin — DISMISSED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**